jsonrpc/debug_endpoint.go

```diff
@@ -843,6 +927,50 @@ func (d *Debug) GetAccessibleState(from, to BlockNumber) (interface{}, error) {
 	)
 }
 
+// ChaindbProperty returns leveldb properties of the key-value database.
+func (d *Debug) ChaindbProperty(property string) (interface{}, error) {
```

Comment:
ChaindbProperty doesn't have a param as an input. It returns leveldb stats. Stat() method in ethdb/leveldb/leveldb.go.

Reply:
Oh, I’m not following. I need to think. This is the final code:

// Stat returns a particular internal stat of the database.
func (kv *KVStorage) Stat(property string) (string, error) {
	return kv.db.GetProperty(property)
}

Reply:
Try to test what happens if you send "" as a property name. In that case you want to return DB stats, which is correct, but LevelDB has a method Stat() to return all stats. I am not sure it will work like this. If we can make it working to return all properties (stats) when sending "" and to return only 1 property when sending property name then keep it as it is. Geth has a different implementation to always return Stat() and there is no param property in ChaindbProperty() method, but as said we can go with your implementation if it works ok for all cases :)

Reply:
![image](https://github.com/user-attachments/assets/6e6f0680-08c8-4a8b-a9dd-fa2f560ecdae)
It works, I hope and as expected :)